_scripts/make.go

```diff
@@ -110,6 +113,36 @@ This option can only be specified if testset is basic or a single package.`)
 	return RootCommand
 }
 
+// copyDAPBinary replicates the dlv installation into dlv-dap binary
```

Comment:
I think this should make a symlink with `os.Link` instead of copying it.

Reply:
Done